app/magento2/base.yml

```diff
@@ -132,6 +147,7 @@ app:
 
     db:
       $ref: /service/mysql/5.7
+      image: mariadb:10.6
```

Comment:
Why did you add this, this seems unrelated?

Reply:
Magento2 didn't work for me using the older version used by Magento1, so I only changed it here.